ToolBox/IQFeed/IQ/IQLookupHistorySymbolClient.cs

```diff
@@ -284,13 +282,16 @@ public int RequestTickData(string symbol, int days, bool oldToNew, Time timeStar
         public int RequestTickData(string symbol, DateTime start, DateTime? end, bool oldToNew, Time timeStartInDay = null, Time timeEndInDay = null)
         {
             _lastRequestNumber++;
-            var reqNo = LookupType.REQ_HST_TCK.ToString() + _lastRequestNumber.ToString("0000000");
+            var reqNo = $"{LookupType.REQ_HST_TCK}{_lastRequestNumber.ToStringInvariant("0000000")}";
             //if (timeStartInDay == null) timeStartInDay = _timeMarketOpen;
             //if (timeEndInDay == null) timeEndInDay = _timeMarketClose;
 
-            var reqString = string.Format("HTT,{0},{1},{2},{3},{4},{5},{6},{7},{8}\r\n", symbol, start.ToString("yyyyMMdd HHmmss"),
-                end.HasValue ? end.Value.ToString("yyyyMMdd HHmmss") : "", _histMaxDataPoints.ToString("0000000"),
-                timeStartInDay == null ? "" : timeStartInDay.IQFeedFormat, timeEndInDay == null ? "" : timeEndInDay.IQFeedFormat, oldToNew ? "1" : "0", reqNo, _histDataPointsPerSend.ToString("0000000"));
+            var reqString = $"HTT,{symbol},{start.ToStringInvariant("yyyyMMdd HHmmss")}," +
+                $"{(end.HasValue ? end.Value.ToStringInvariant("yyyyMMdd HHmmss") : "")}," +
+                $"{_histMaxDataPoints.ToStringInvariant("0000000")}," +
+                $"{(timeStartInDay == null ? "" : timeStartInDay.IQFeedFormat)}," +
+                $"{(timeEndInDay == null ? "" : timeEndInDay.IQFeedFormat)},{(oldToNew ? "1" : "0")}," +
+                $"{reqNo},{_histDataPointsPerSend.ToStringInvariant("0000000")}\r\n";
```

Comment:
Not a required change for now, but I think using the `Invariant()` extension method instead of multiple `ToStringInvariant()` calls seems to make these long interpolated strings shorter and more readable. 
Thoughts?